internal/osutils/shortcut/shortcut_windows.go

```diff
@@ -96,6 +96,20 @@ func (s *Shortcut) setIcon(path string) error {
 	return nil
 }
 
+func (s *Shortcut) SetMinimized() error {
+	_, err := oleutil.PutProperty(s.dispatch, "WindowStyle", 7)
```

Comment:
Do we have to use this magic number? Is there a constant somewhere we can make use of? If not, please add a comment with what it means and/or the enum used by Windows/OLE (e.g. `// VK_MINIMIZE`, which I just made up)

Reply:
It is essentially a magic number. Good point though I will try to make this more clear.